tests/enip-stats-udp/test.yaml

```diff
@@ -0,0 +1,28 @@
+requires:
+  min-version: 8
+
+pcap: ../enip-alert/enip_test1.pcap
+
+# disables checksum verification
+args:
+- -k none --set stream.midstream=true
+
+checks:
+  - filter:
+      count: 1
+      match:
+        event_type: flow
+        app_proto: enip
+  - filter:
```

Comment:
I don't understand this. If we have a flow with enip, shouldn't we also have a non-0 stats entry?

Reply:
Yes, but that is another bug.

There are many cases where the stats do not reflect the number of flows

`jq 'select(.event_type=="flow" and .app_proto=="ftp") | .app_proto'  log/eve.json | wc -l`
And
`jq 'select(.event_type=="stats") | .stats."app_layer".flow.ftp' log/eve.json`
do not give the same number

cf https://github.com/OISF/suricata/pull/8328